server/db.js

```diff
@@ -2,12 +2,14 @@ import pg from "pg";
 
 const { Pool } = pg;
 
+
+
 let localPoolConfig = {
   user: "postgres",
   password: "Brenda@919",
   host: "localhost",
   port: "5432",
-  database: "freementors",
+  database: "postgres",
```

Comment:
Put these in an environment variable.

Reply:
I am not able to put the user and password in an ENV

I get these errors:
The db user changed to my PC user
while for Password: SASL: SCRAM-SERVER-FIRST-MESSAGE: client password must be a string

Reply:
have you tried googling the error?